canonical-websites/www.ubuntu.com
# Mongoose no more!

## Done

* s/mongoose/www/ in all forms return URLs
* Ensure all **global nav** URLs are ~1) absolute~ 2) not pointing to mongoose

### Drive-by
* Fixed blank `/ai` link in developer menu

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/](http://0.0.0.0:8001/)
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)
- Ensure links in global nav take you to www.ubuntu.com, not mongoose.ubuntu.com
- Ensure a form submission takes you to a "thank you" page on www.ubuntu.com, not on mongoose.ubuntu.com
